### PR TITLE
Update readme with correct version and logs section

### PR DESCRIPTION
**Pull request by sarah-witt**
### Requirements for Contributing to this repository

* Fill out the template below. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* The pull request must only fix one issue at the time.
* The pull request must update the test suite to demonstrate the changed functionality.
* After you create the pull request, all status checks must be pass before a maintainer reviews your contribution. For more details, please see [CONTRIBUTING](/CONTRIBUTING.md). 

### What does this PR do?

- Updates some wording in the readme
- Creates a log collection section

### Review checklist (to be filled by reviewers)

- [ ] Feature or bug fix MUST have appropriate tests (unit, integration, etc...)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have one `changelog/` label attached. If applicable it should have the `backward-incompatible` label attached.
- [ ] PR should not have `do-not-merge/` label attached.
- [ ] If Applicable, issue must have `kind/` and `severity/` labels attached at least.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,33 +8,24 @@ A Jenkins plugin for automatically forwarding metrics, events, and service check
...
 **Note**: If you see an unexpected version of the `Datadog Plugin`, run `Check Now` from the `Manage Jenkins -> Manage Plugins` screen.
 
 ### Configuration
 
-You can use two ways to configure your plugin to submit data to Datadog:
+There are two ways to configure your plugin to submit data to Datadog:
 
 * **RECOMMENDED**: Using a DogStatsD server / Datadog Agent that acts as a forwarder between Jenkins and Datadog.
-  - Build Logs collection only works with a full Datadog Agent installed.
+  - [Build Logs collection](#log-collection) only works with a full Datadog Agent installed.
```
Comment (FlorianVeaux) [inline README.md:28]: I don't think that's true, the http client forwards logs

File: README.md [hunk 2]
```diff
@@ -238,6 +229,31 @@ NOTE: `event_type` is always set to `security` for above events and metrics.
 | `jenkins.user.authenticated`           | Rate of users authenticating.                                  | `jenkins_url`, `user_id`                                    |
 | `jenkins.user.logout`                  | Rate of users logging out.                                     | `jenkins_url`, `user_id`                                    |
 
+
+#### Log Collection
+
+**Note**: This configuration only applies to those using the [Datadog Agent configuration](#dogstatsd-forwarding-plugin).
```
Comment (FlorianVeaux) [inline README.md:235]: We should also document how to setup log collection with the http client.

File: README.md (hunk 1 above)
Comment (sarah-witt) [reply README.md:28]: True, I will update with that!

Comment (sarah-witt) [reply README.md]: Updated!

File: README.md [hunk 3]
```diff
@@ -238,6 +237,31 @@ NOTE: `event_type` is always set to `security` for above events and metrics.
...
+2. To collect Jenkins logs, create a [custom log source file][13] for your Agent by creating a `conf.yaml` inside `conf.d/jenkins.d` with the following:
+    ```
+    logs:
+      - type: tcp 
+        port: <PORT> 
+        service: <SERVICE>
+        source: jenkins
+    ```
+    
+3. In Jenkins, submit the port you specified above as the `Log Collection Port`. You can set this using [env vars](#dogstatsd-forwarding-env), a [groovy script](#dogstatsd-forwarding-groovy-script), or the [Jenkins UI](#dogstatsd-forwarding-plugin)
```
Comment (sarina-dd) [inline README.md:261]:
```suggestion
3. In Jenkins, submit the port you specified above as the `Log Collection Port`. You can set this using [env vars](#dogstatsd-forwarding-env), a [groovy script](#dogstatsd-forwarding-groovy-script), or the [Jenkins UI](#dogstatsd-forwarding-plugin).
```

**Pull request merged**